src/gui/MainWindow.cpp

```diff
@@ -983,7 +986,7 @@ void MainWindow::hideWindow()
 
 void MainWindow::toggleWindow()
 {
-    if ((QApplication::activeWindow() == this) && isVisible() && !isMinimized()) {
```

Comment:
I'm afraid this will cause undesirable effects on MacOS

Reply:
@droidmonkey Can you elaborate? A priori I can't see it causing problems, since it looks like an old check (it was there before the other two conditions were implemented), it works on Windows and Linux, and it wasn't added specifically for Mac, though it would be nice for someone to check.

Reply:
I'm just saying that we edited it and thoroughly tested on Linux and Windows but not Mac. What I meant to say is "this needs to be tested on Mac to ensure no undesirable effects"

Reply:
@droidmonkey I tested yesterday and saw no problems toggling the window from the tray icon on Mac.